src/FritzBox/Converter.php

```diff
@@ -192,8 +192,8 @@ private function getPhoneNumbers(stdClass $card): array
                         unset($card->xquickdial);                          // flush used quick dial number
                     }
                 } else {
-                    $format = "The quick dial number >%s< has been assigned more than once (%s)!";
-                    error_log(sprintf($format, $card->xquickdial, $number));
+                    $format = "The quick dial number >%s< has been assigned more than once!";
```

Comment:
remove format variable, just include string in sprintf

Reply:
This was "old" code not done by me - I just fixed the existing phpstan error.
But OK - can do it.

Reply:
Think I've changed that in #117- I'm afraid you'll have to rebase your PR once that is in (blacksenator is testing it today).